lib/Date.php

```diff
@@ -8,11 +8,11 @@
 
 final class Date
 {
-    private int $year;
+    private readonly int $year;
 
-    private int $month;
+    private readonly int $month;
 
-    private int $day;
+    private readonly int $day;
```

Comment:
Is there any reason this was not moved to ctor property promotion? :see_no_evil:

Reply:
No, it's just what Rector did, I'll update.

Reply:
https://github.com/Lendable/clock/pull/427/commits/f500c5274499bec49736c3ee1aaa539f70ec47b0